releasenotes/notes/no-useless-DNS-queries-1ed8ff14e0684da1.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Make the agent use FQDN when building intake hostnames to avoid
+    generating as many useless DNS queries as entries in the
+    ``search`` section of the ``/etc/resolv.conf` file.
```

Comment:
```suggestion
    Use FQDNs when the Agent builds intake hostnames to prevent generating as many DNS queries as there are entries in the ``search`` section of the ``/etc/resolv.conf` file.
```

Reply:
Maybe specify that it's only when DD_SITE config is used, not when the full URL is provided ?